.gitignore

```diff
@@ -1,2 +1,3 @@
+.vscode/
```

Comment:
I don't think we actually want this file ignored - can you help me understand why this was added?

Reply:
You're right, I got confused thinking I would upload things from my .vscode config.